### PR TITLE
[ADF-2304] Add option to Content Node Selector to transform the breadcrumb folder node

### PR DESCRIPTION
**Pull request by suzanadirla**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Please see Jira task https://issues.alfresco.com/jira/browse/ADF-2304

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by eromano** [comment]
missing documentation

File: docs/content-node-selector-panel.component.md [hunk 1]
```diff
@@ -25,12 +25,14 @@ Opens a [Content Node Selector](content-node-selector.component.md) in its own d
 | rowFilter | RowFilter | null | Custom row filter function. [See More](document-list.component.md#custom-row-filter) |
 | imageResolver | ImageResolver | null | Custom image resolver function. [See More](document-list.component.md#custom-image-resolver) |
 | pageSize | number | 10 | Number of items shown per page in the list |
+| isSelectionValid | ValidationFunction | defaultValidation | Function used to decide if the selected node has permission to be the chosen. The defaultValidation always returns true.  |
+| breadcrumbTransform | (node) => any | null | Action to be performed to the chosen/folder node before building the breadcrumb UI. Can be useful in case a custom formatting is needed to the breadcrumb, so changing the node's path elements that help build the breadcrumb can be done through this function. |
```
Comment (eromano) [inline docs/content-node-selector-panel.component.md]: is it possible have also an example with an image?

Comment (suzanadirla) [reply docs/content-node-selector-panel.component.md]: I added an example to the documentation file, please check. Thank you.

Comment (eromano) [reply docs/content-node-selector-panel.component.md]: thanks

**Pull request merged**